src/components/AddressSearch.js

```diff
@@ -79,55 +82,66 @@ const AddressSearch = (props) => {
     };
 
     return (
-        <GooglePlacesAutocomplete
-            ref={googlePlacesRef}
-            fetchDetails
-            suppressDefaultStyles
-            enablePoweredByContainer={false}
-            onPress={(data, details) => saveLocationDetails(details)}
-            query={{
-                key: 'AIzaSyC4axhhXtpiS-WozJEsmlL3Kg3kXucbZus',
-                language: props.preferredLocale,
-                types: 'address',
-                components: 'country:us',
-            }}
-            requestUrl={{
-                useOnPlatform: 'web',
-                url: `${CONFIG.EXPENSIFY.URL_EXPENSIFY_COM}api?command=Proxy_GooglePlaces&proxyUrl=`,
-            }}
-            textInputProps={{
-                InputComp: ExpensiTextInput,
-                label: props.label,
-                containerStyles: props.containerStyles,
-                errorText: props.errorText,
-                onChangeText: (text) => {
-                    const isTextValid = !_.isEmpty(text) && _.isEqual(text, props.value);
 
-                    // Ensure whether an address is selected already or has address value initialized.
-                    if (!_.isEmpty(googlePlacesRef.current.getAddressText()) && !isTextValid) {
-                        saveLocationDetails({});
-                    }
-                },
+        // We use the View height to determine if we should hide the border and margin of the listView dropdown
+        // to prevent a lingering border when there are no address suggestions.
+        // The height of the input + wrapped error message is 112 pixels
+        <View
+            onLayout={(event) => {
+                const {height} = event.nativeEvent.layout;
+                return height > 112 ? setDisplayListViewBorder(true) : setDisplayListViewBorder(false);
```

Comment:
I don't know the answer to this question but it seems worth asking. Can we use a more dynamic way to measure the height? We have a ref to the `<GooglePlacesAutoComplete>` component so maybe we can use the `measure` function (mentioned [here](https://github.com/facebook/react-native/issues/953)).

Reply:
Hmm we can definitely get the input's height dynamically. I'm not sure if it's worth it though, because we still have to manually compensate for the top margin and border width, so the conditional becomes something like this `height > inputHeight + 6`. Also, the solution is far from elegant 🤣

Reply:
Actually managed to move the onLayout prop to the `GooglePlacesAutocomplete` component, so now the solution is independent of the text input height. Thanks for the tip @Luke9389!

Reply:
Actually managed to move the onLayout prop to the `GooglePlacesAutocomplete` component and now the solution is independent of the text input height. Thanks for the tip @Luke9389!

Reply:
Cool, I'll review again

Reply:
Oh rad, nice work @luacmartins! I really appreciate you following through on that. 🙇